### PR TITLE
3.38.1 customer replica

### PR DESCRIPTION
**Pull request by camdencheek**
Merge 3.38.1 patch commits into 3.38 branch

### Checklist

* [ ] Sister [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph) change:
* [ ] All images have a valid tag and SHA256 sum

### Test plan

N/A

### CODE REVIEW

File: pure-docker/deploy.sh [hunk 1]
```diff
@@ -10,7 +10,6 @@ source ./replicas.sh
 
 docker network create sourcegraph &> /dev/null || true
 
-./deploy-apache.sh
```
Comment (caugustus-sourcegraph) [inline pure-docker/deploy.sh:13]: Whoops, all these changes need to be reverted

**Review by caugustus-sourcegraph — CHANGES_REQUESTED** [review]

File: pure-docker/deploy.sh (hunk 1 above)
Comment (camdencheek) [reply pure-docker/deploy.sh:13]: Thanks for catching that. Looks like we should update the docs 😄

**Comment by camdencheek** [comment]
@caugustus-sourcegraph I can't seem to force-push to the `3.38.1-customer-replica` branch. Can branch protections be relaxed for that?

**Comment by caugustus-sourcegraph** [comment]
@camdencheek - granted you force-push permissions, let me know if it doesn't work

**Comment by camdencheek** [comment]
@caugustus-sourcegraph No luck. Still getting branch protection errors from Github

**Comment by camdencheek** [comment]
Whoops, didn't mean to re-request review yet. I'm unable to force push still

**Comment by camdencheek** [comment]
It worked! Should be ready for re-review

**Review by caugustus-sourcegraph — APPROVED** [review]

**Pull request merged**